pyqir-evaluator/NOTICE-WHEEL.txt

```diff
@@ -9103,7 +7345,7 @@ SOFTWARE.
 
 
 
-pyqir-evaluator 0.7.0a1 - SPDX: MIT - MIT License
+pyqir-evaluator 0.7.0-alpha.1 - SPDX: MIT - MIT License
```

Comment:
Version changed from `pyqir-evaluator 0.7.0a1 - SPDX: MIT - MIT License` to `pyqir-evaluator 0.7.0-alpha.1 - SPDX: MIT - MIT License`. Was this intentional?

Reply:
I don't know why it changed, but the notice files are up-to-date.